src/java.base/unix/native/libnio/fs/UnixNativeDispatcher.c

```diff
@@ -247,6 +247,20 @@ static int fstatat_wrapper(int dfd, const char *path,
 }
 #endif
 
+/**
+ * Lookup time functions symbols, trying 64 bit version first
+ */
+static void* lookup_time_function(const char* symbol64, const char* symbol) {
```

Comment:
This function is not specific to "time functions" so will need to be renamed.

Reply:
There is a check for time_t size inside it. Maybe `lookup_time_t_function` ?